starknet_py/net/client_models.py

```diff
@@ -618,27 +598,32 @@ class StorageDiffItem:
 
 @dataclass
 class EstimatedFee:
+    # pylint: disable=too-many-instance-attributes
     """
     Dataclass representing estimated fee.
     """
 
-    gas_consumed: int
-    gas_price: int
-    data_gas_consumed: int
-    data_gas_price: int
+    l1_gas_consumed: int
+    l1_gas_price: int
+    l2_gas_consumed: int
+    l2_gas_price: int
+    l1_data_gas_consumed: int
+    l1_data_gas_price: int
     overall_fee: int
     unit: PriceUnit
 
+    # TODO (#1498): Decrease multipliers
     def to_resource_bounds(
         self, amount_multiplier=1.5, unit_price_multiplier=1.5
     ) -> ResourceBoundsMapping:
```

Comment:
`ResourceBoundsMapping` needs to be updated to include [l1_data_gas](https://github.com/starkware-libs/starknet-specs/blob/v0.8.0/api/starknet_api_openrpc.json#L3631)